### ISSUE TITLE
Add Jenkins badge for testing examples to `README.md`

### ISSUE THREAD

**Issue by jchodera** [choderalab/yank#381]
To catch failing examples, we should add a Jenkins badge to indicate whether the current examples are passing or failing:

[![Build Status](https://jenkins.choderalab.org/buildStatus/icon?job=test-yank-linux-xeon-gtxtitan-jak)](https://jenkins.choderalab.org/job/test-yank-linux-xeon-gtxtitan-jak/)

Source:
```
[![Build Status](https://jenkins.choderalab.org/buildStatus/icon?job=test-yank-linux-xeon-gtxtitan-jak)](https://jenkins.choderalab.org/job/test-yank-linux-xeon-gtxtitan-jak/)
```
I just realized that there were a few things broken with the Jenkins example test configuration. I've hopefully fixed those now!

**Comment by jchodera** [comment]
That's odd---the `test.sh` files have disappeared from the examples.  We'll need to add these back to get the automated example testing to run, @andrrizzi.  These should be like `run.sh`, but should ideally run a shorter simulation that simply ensures things are working properly.

**Comment by jchodera** [comment]
Fixed by #386
